tests/integration_tests/config/generate_simtel_event_data_hdf5.yml

```diff
@@ -4,7 +4,7 @@ CTA_SIMPIPE:
   - APPLICATION: simtools-generate-simtel-event-data
     TEST_NAME: hdf5
     CONFIGURATION:
-      INPUT: 'tests/resources/*proton*.simtel.zst'
+      INPUT: 'tests/resources/*proton*North_alpha_*.simtel.zst'
```

Comment:
Is this a good idea to have wildcards here? In the future you might have more files which fit it and run this test all the time on them without meaning to.

Reply:
Fixed.